packages/ui/Touchable/Touchable.js

```diff
@@ -521,6 +526,13 @@ const Touchable = hoc(defaultConfig, (config, Wrapped) => {
 			return true;
 		}
 
+		onBlur () {
+			this.hold.suspend();
+			this.drag.end();
```

Comment:
Would it make sense to call the same handler as `onMouseLeave` calls here instead of trying to handle it differently?